embassy-mcxa/Cargo.toml

```diff
@@ -48,6 +48,9 @@ maitake-sync = { version = "0.2.2", default-features = false, features = ["criti
 embassy-time = { version = "0.5.0", optional = true }
 embassy-time-driver = { version = "0.2.1", optional = true }
 
+rand-core-06 = { package = "rand_core", version = "0.6" }
```

Comment:
Any reason to support 0.6? It seems like the bulk of the world is already on v0.9.x: https://crates.io/crates/rand-core